frontend/components/dashboard/table/DashboardTableRewards.vue

```diff
@@ -208,13 +148,8 @@ const findNextEpochDuties = (epoch: number) => {
               :header="$t('dashboard.validator.col.duty')"
             >
               <template #body="slotProps">
-                <span
-                  v-if="slotProps.data.group_id === DAHSHBOARDS_NEXT_EPOCH_ID"
-                >
-                  {{ findNextEpochDuties(slotProps.data.epoch) }}
-                </span>
                 <DashboardTableValueDuty
-                  v-else
+                  :is-number-visible="slotProps.data.group_id !== DAHSHBOARDS_NEXT_EPOCH_ID"
```

Comment:
np: but `number` is also very `generic` here
```suggestion
                  :has-number="slotProps.data.group_id !== DAHSHBOARDS_NEXT_EPOCH_ID"
```

Reply:
done